lessons/advanced/strings-in-depth.md

```diff
@@ -0,0 +1,156 @@
+---
+layout: page
+title: Strings in-depth 
```

Comment:
@setzer777 / @pjhampton — What about just "Strings" and adding this to the Basics section instead of Advanced?